# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasink/moz.build
# Commit: c57c41e8c39e
# Full Hash: c57c41e8c39ea51e96c13af8ecdcbe1640e2a9d6
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-11-14 17:12:31
# Regressor Bug: 1172394
# File Overlap Count: 1
# Description:
#   Bug 1172394 - Refactor how DecodedStream is set up. r=padenot
#   
#   This patch removes the responsibility of js-facing MediaStreamTracks from the
#   MediaDecoder stack, and moves the machinery for setting up DecodedStream to
#   higher order functions like state mirroring and watchables.
# ==============================================================================

diff -r a796541fe5ef -r c57c41e8c39e dom/media/mediasink/moz.build
--- a/dom/media/mediasink/moz.build	Wed Nov 13 08:55:29 2019 +0000
+++ b/dom/media/mediasink/moz.build	Wed Nov 13 08:55:39 2019 +0000
@@ -8,7 +8,6 @@
     'AudioSink.cpp',
     'AudioSinkWrapper.cpp',
     'DecodedStream.cpp',
-    'OutputStreamManager.cpp',
     'VideoSink.cpp',
 ]
 
